aws/adapter.go

```diff
@@ -400,20 +400,20 @@ func (a *Adapter) UpdateTargetGroupsAndAutoScalingGroups(stacks []*Stack) {
 	for _, asg := range a.autoScalingGroups {
 		// This call is idempotent and safe to execute every time
 		if err := updateTargetGroupsForAutoScalingGroup(a.autoscaling, a.elbv2, targetGroupARNs, asg.name, ownerTags); err != nil {
-			log.Printf("UpdateTargetGroupsAndAutoScalingGroups() failed to attach target groups to ASG: %v", err)
+			log.Debugf("UpdateTargetGroupsAndAutoScalingGroups() failed to attach target groups to ASG: %v", err)
```

Comment:
I think this should be an error log instead of debug because this could indicate something is wrong.